Index.html

```diff
@@ -26,244 +27,263 @@
+      <img src="./picture/Snapshoot Portfoliopc.png" alt="">
+    </div>
+  </div>
+  
+  <!-- third page -->
+  
+  <div class="repeated-div">
+    <img
+      class="repeated-img"
+      src="./picture/Snapshoot Portfolio3.png"
+      alt=""
+    />
+    <img class="pcrepeated-img" src="./picture/pc3.png" alt="">
+    <div class="pc12-div">
+      <h2 class="repeated-p1">Tonic</h2>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+    
+    <button class="see-button">See project</button>
+  </div>
+  </div>
+  <!-- four page -->
+  
+  <div class="repeated-div-2">
+    <img
+      class="repeated-img"
+      src="./picture/Snapshoot Portfolio2.png"
+      alt=""
+    />
+<div>
+  <h2 class="repeated-p1">
+    Multi-Post Stories
+    
+  </h2>
+  <ul class="repeated-ul">
+    <li class="repeated-li1">CANOPY</li>
+    <li class="repeated-li2">Back End Dev</li>
+    <li class="repeated-li3">2015</li>
+  </ul>
+  <p class="repeated-p2">
+    A daily selection of privately personalized reads; no accounts or
+    sign-ups required.
+  </p>
+  <ul class="repeated-ul2">
+    <li class="repeated-li4">HTML</li>
+    <li class="repeated-li5">CSS</li>
+    <li class="repeated-li6">JavaScript</li>
+  </ul>
+  <button class="see-button">See project</button>
+</div>
+    <div class="second-page-image">
+      <img src="./picture/pc4.png" alt="">
+    </div>
+  </div>
 
     <!-- about myself -->
 
     <section class="about-myself">
       <div class="div-flex">
-        <h2 class="About-name">About Myself</h2>
-        <p class="intro">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website. Look through some of my work and experience! If
-          you like what you see and have a project you need coded, don’t
-          hesitate to contact me.
-        </p>
-        <h3 class="Connect">LET'S CONNECT</h3>
-        <ul class="social-1">
-          <li>
-            <a
-              class="linkedin"
-              href="https://www.linkedin.com/in/kerlos-bahaa-457a8925a/"
-            >
-              <i class="fa-brands fa-linkedin-in"></i>
-            </a>
-          </li>
-          <li>
-            <a href="#"> <img src="./picture/Vector.png" alt="" /></a>
-          </li>
-          <li>
-            <a class="github" href="https://github.com/kerlos-bahaa">
-              <i class="fa-brands fa-github"></i>
-            </a>
-          </li>
+  <div class="aboutmyself-top">
+    <h2 class="About-name">About Myself</h2>
```

Comment:
- [ ] please kindly add a little padding left to the about section so it matches the original design

![Screenshot (916)_LI](https://user-images.githubusercontent.com/65298048/228686323-8fa56acf-9235-4824-858b-171c617490e3.jpg)